src/power/lid.rs

```diff
@@ -63,14 +63,11 @@ impl LidRetriever for Lid {
 
         let lid_str = fs::read_to_string(self.best_path)?;
 
-        let state = if lid_str.contains("open") {
-            LidState::Open
-        } else if lid_str.contains("closed") {
-            LidState::Closed
-        } else {
-            LidState::Unknown
-        };
-        Ok(state)
+        Ok(match lid_str.split_whitespace().last().unwrap() {
```

Comment:
Would the unwrap crash acs if the lid file wasn't in the right file or doesn't exist?

Reply:
It might be a safe assumption that it's in the correct format

Reply:
wait actually, could you do unwrap or error and set state = LidState::Unknown if it doesn't unwrap correctly otherwise return the state

Reply:
It does not crash. The if statement before prevents the crash.

```rs
        if !self.found_path {
            return Ok(LidState::Unapplicable);
        }
```